docs/api/bot.rst

```diff
@@ -83,6 +83,13 @@ components.
       The :py:class:`botogram.User` representation of the bot's user account.
       From this you can access its id, username and more.
 
+   .. py:attribute:: override_i18n
+
+      A dictionary that allows to override default i18n messages by associating
+      the default ``msgid`` string of a message with its alternative.
+
+      .. versionadded:: 0.4.1
```

Comment:
Since it's a new feature it will probably will be added in 0.5

Reply:
Yep, but I'll tweak the changelog myself after merging, don't worry.